packages/scoutgame-ui/src/providers/PurchaseProvider.tsx

```diff
@@ -145,6 +144,7 @@ export function PurchaseProvider({ children }: { children: ReactNode }) {
         {
           onSuccess: async (_data) => {
             setPurchaseSuccess(true);
+            toast.info('NFT purchase is sent and will be confirmed shortly');
```

Comment:
we already show the user an error inline in the form when sendMintTransaction fails, so i dont think we need a message here